contrib/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# example of Dockerfile that builds release of electrumx-1.13.0
+# ENV variables can be overrided on the `docker run` command
+
+FROM ubuntu:18.04
+
+WORKDIR /
+ADD https://github.com/kyuupichan/electrumx/archive/1.13.0.tar.gz /
+RUN tar zxvf *.tar.gz
+
+RUN apt-get update && \
+        apt-get -y install python3.7 python3-pip librocksdb-dev libsnappy-dev libbz2-dev libz-dev liblz4-dev && \
+        pip3 install aiohttp pylru python-rocksdb
+
+RUN cd /electrumx* && python3 setup.py install
+
+ENV SERVICES="tcp://:50001"
+ENV COIN=BitcoinSV
+ENV DB_DIRECTORY=/db
+ENV DAEMON_URL="http://username:password@hostname:port/"
+ENV ALLOW_ROOT=true
+ENV DB_ENGINE=rocksdb
+ENV MAX_SEND=10000000
+ENV BANDWIDTH_UNIT_COST=50000
+ENV CACHE_MB=2000
+
+VOLUME /db
+
+RUN mkdir -p "$DB_DIRECTORY" && ulimit -n 1048576
+
+CMD ["/usr/bin/python3", "/usr/local/bin/electrumx_server"]
```

Comment:
Hmm shouldn't be /usr/bin/python3.7 instead of /usr/bin/python3?

Reply:
Ubuntu distro has a symlink to their python 3.* installation.